src/components/CallView/shared/Video.vue

```diff
@@ -225,22 +225,53 @@ export default {
 			return peerData
 		},
 
+		participant() {
```

Comment:
reuse or integrate into the participant mixin ?

note that the mixin would return the disconnected flag instead of just an empty object

Reply:
participant mixin is always the current user, this here is peerId, not yourself.